index.html

```diff
@@ -0,0 +1,1555 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title class="display-1">NEWSWEK</title>
+    <!-- bootstrap -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="font-awesome-4.7.0/font-awesome-4.7.0/css/font-awesome.min.css">
+    <!-- fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Merriweather&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Oswald&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+    <!-- style.css -->
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/reset.css">
+</head>
+<body>
+  <header>
+    <div class=" nav1 container-lg d-flex justify-content-between w-100  mx-1 overflow-hidden" >
+      <nav class="d-flex flex-column navbar px-0 mx-0 w-100">
+        <div class="meteo-date d-lg-flex flex-lg-column text-white d-md-none d-sm-none">
+          <div class="meteo d-flex flex-row">
+            <div class="cloud">
+              <i class="fas fa-cloud"></i>
+            </div>
+            <div class="temperature">
+              <h6> 28°Bujumbura, BI </h6>
+            </div>
+          </div>
+          <div class="left-margin-10">
+            <div class="time"> Fri, Jun 19, 2020</div>
+          </div>
+        </div>
+        <div class="logo col-4 col-lg-4">
+         <h2><a class="text-white" href="#"> newsweek</a></h2>
+        </div>
+        <div class="right-hand d-flex flex-row align-items-center px-2 ml-lg-6">
+          <div class="login">
+            <a class="d-md-none d-lg-none" href="#"><img src="images/user.png" class="user" alt="user"> </a>
+            <button class="border-0 font-weight-bold color-white d-sm-flex- d-none d-md-flex mt-lg-3">login</button>
+          </div>
+          <div class="subscribe ml-2 text-white mt-lg-3">
+            <button class="rounded border-0 ">SUBSCRIBE</button>
+            
+          </div>
+          <div class="menu-svg mr-3 d-lg-none">
+            <img src="images/menu.svg" alt="menu-icon">
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div class="container-fluid nav2 d-lg-flex flex-row w-100 d-md-none d-sm-none">
+      <ul class="d-flex flex-row justify-content-between align-items-center w-100 list-unstyled px-3">
+        <li>.u.s.</li>
+        <li>world</li>
+        <li>business</li>
+        <li>tech & science</li>
+        <li>culture</li>
+        <li>newsgeeks</li>
+        <li>sports</li>
+        <li>health</li>
+        <li>the debate</li>
```

Comment:
center the text inside the li's, cause they are not centered, you can use text-center:
![image](https://user-images.githubusercontent.com/34653764/85727920-57851580-b6ef-11ea-8880-4e06f844be76.png)